frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -167,10 +167,10 @@ export const TARGET_AREA_TO_NAME = [
 ]
 
 export const SEVERITY_LEVEL_TO_NAME = {
-    critical: 'Product outage / data loss / data breach',
-    high: 'Specific feature not working at all',
-    medium: 'Feature functioning but not as expected',
-    low: 'General question or feature request',
+    critical: 'Critical: Outage, data loss, or data breach',
+    high: 'High: Specific feature is not working at all',
+    medium: 'Medium: Feature not working as expected',
+    low: 'Low: Question or feature request',
```

Comment:
```suggestion
    critical: 'Outage, data loss, or data breach',
    high: 'Specific feature is not working at all',
    medium: 'Feature not working as expected',
    low: 'Question or feature request',
```

Let's not show the actual severity here, since that caused people to abuse this selection, and select how important the issue is to them.

Reply:
```suggestion
    critical: 'Outage, data loss, or data breach',
    high: 'Feature is not working at all',
    medium: 'Feature not working as expected',
    low: 'Low: Question or feature request',
```